docs/.vitepress/config/index.ts

```diff
@@ -9,6 +9,7 @@ export default defineConfig({
   locales: {
     root: { label: 'English', ...en },
     zh: { label: '简体中文', ...zh },
-    pt: { label: 'Português', ...pt }
+    pt: { label: 'Português', ...pt },
+    ko: { label: '한국어', lang: 'ko-KR', link: 'https://vitepress.vuejs.kr/' }
```

Comment:
Add this lang to your site's config too, it currently shows `lang="en-US"` there

Reply:
I did it! thx!